src/api/repos/contributors.rs

```diff
@@ -0,0 +1,52 @@
+use super::*;
+
+#[derive(serde::Serialize)]
+pub struct ListContributorsBuilder<'octo, 'r> {
+    #[serde(skip)]
+    handler: &'r RepoHandler<'octo>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    anon: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    per_page: Option<u8>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    page: Option<u32>,
+}
+
+impl<'octo, 'r> ListContributorsBuilder<'octo, 'r> {
+    pub fn new(handler: &'r RepoHandler<'octo>) -> Self {
+        Self {
+            handler,
+            anon: None,
+            per_page: None,
+            page: None,
+        }
+    }
+
+    /// Set to 1 or true to include anonymous contributors in results.
+    pub fn anon(mut self, include_anon: impl Into<bool>) -> Self {
```

Comment:
Dumb question: does it matter that these aren't `&mut self`?

Reply:
Here we consume `self` and then return it, as a part of a builder pattern.

Reply:
That makes sense, sorry I missed it the first go-round.